contracts/src/IRollup.sol

```diff
@@ -220,6 +228,21 @@ interface IRollup {
      */
     function setBaseStakeAmount(uint256 newAmount) external;
 
+    /**
+     * @notice Set the addresses of the validator whitelist
```

Comment:
Can you write a better notice? I'm thinking this interface can be a little unintuitive, in that if an address previously whitelisted, but doesn't have a state update supplied to this function, they will remain whitelisted (that is, they have to be specifically unwhitelisted). Maybe a better function name might clear this up as well? `updateValidators`.

Reply:
pushed the change